code/_globalvars/bitfields.dm

```diff
@@ -566,7 +566,9 @@ GLOBAL_LIST_INIT(bitfields, list(
 		"INTERACT_REQUIRES_DEXTERITY" = INTERACT_REQUIRES_DEXTERITY,
 		"INTERACT_CHECK_INCAPACITATED" = INTERACT_CHECK_INCAPACITATED,
 		"INTERACT_UI_INTERACT" = INTERACT_UI_INTERACT,
-		"INTERACT_SILICON_ALLOWED" = INTERACT_SILICON_ALLOWED
+		"INTERACT_SILICON_ALLOWED" = INTERACT_SILICON_ALLOWED,
+		"INTERACT_POWERLOADER_PICKUP_ALLOWED" = INTERACT_POWERLOADER_PICKUP_ALLOWED,
+		"INTERACT_POWERLOADER_PICKUP_ALLOWED_BYPASS_ANCHOR" = INTERACT_POWERLOADER_PICKUP_ALLOWED_BYPASS_ANCHOR,
```

Comment:
These being interact flags feels very odd considering they dont interact with any of the actual interact flags, whyd you do this?

Reply:
Wasn't sure of a better better flag var to put them in